#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "udpmaster_pkg"


gen = ParameterGenerator()

gen.add("robot1_x", double_t, 0, "Robot1 initial x", 0, -5, 5)
gen.add("robot1_y", double_t, 0, "Robot1 initial y", 0, -5, 5)
gen.add("robot1_yaw", double_t, 0,
        "Robot1 initial yaw; Currently unused", 0, -3.1416, 3.1416)
gen.add("robot2_x", double_t, 0, "Robot2 initial x", 0, -5, 5)
gen.add("robot2_y", double_t, 0, "Robot2 initial y", 1, -5, 5)
gen.add("robot2_yaw", double_t, 0,
        "Robot2 initial yaw; Currently unused", 0, -3.1416, 3.1416)
gen.add("robot3_x", double_t, 0, "Robot3 initial x", 0, -5, 5)
gen.add("robot3_y", double_t, 0, "Robot3 initial y", -1, -5, 5)
gen.add("robot3_yaw", double_t, 0,
        "Robot3 initial yaw; Currently unused", 0, -3.1416, 3.1416)

exit(gen.generate(PACKAGE, "master", "initialPose"))
